src/tracing/service/tracing_service_impl.cc

```diff
@@ -4192,6 +4228,9 @@ base::Status TracingServiceImpl::FlushAndCloneSession(
         args.clone_trigger_producer_name,
         args.clone_trigger_trusted_producer_uid, args.clone_trigger_delay_ms};
   }
+  if (args.output_file_fd) {
```

Comment:
Linked to the other comment, this should not happen if we didn't need a fd because the original session wasn't clone into file